hw01_hello_otus/go.mod

```
@@ -1,3 +1,18 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/Tiksy1/otus_hw-test/tree/master/hw01_hello_otus
```

```suggestion
module github.com/Tiksy1/otus_hw-test/hw01_hello_otus
```
Корректное имя модуля содержит полный путь до репозитория и вложенные директории (если нужно), остальные сегменты с ветками - UI GitHub